src/client/Microsoft.Identity.Client.Broker/WamAdapters.cs

```diff
@@ -268,5 +271,48 @@ private static string GetExpectedRedirectUri(string clientId)
         {
             return $"ms-appx-web://microsoft.aad.brokerplugin/{clientId}";
         }
+
+        /// <summary>
+        /// Converts to MSAL Account Id or Null
+        /// </summary>
+        /// <param name="wamAccount"></param>
+        /// <param name="clientID"></param>
+        /// <param name="logger"></param>
+        public static IAccount ConvertToMsalAccount(
+                NativeInterop.Account wamAccount, 
+                string clientID,
+                ILoggerAdapter logger)
+        {
+            IAccount runtimeAccount;
+
+            try
+            {
+                if (wamAccount.AccountId == null ||
+                    wamAccount.HomeAccountid == null ||
+                    wamAccount.Environment == null ||
+                    wamAccount.UserName == null)
+                {
+                    logger.Info($"[WamBroker] wamAccount.AccountId: {wamAccount.AccountId}.");
+                    logger.Info($"[WamBroker] wamAccount.HomeAccountid: {wamAccount.HomeAccountid}.");
+                    logger.Info($"[WamBroker] wamAccount.Environment: {wamAccount.Environment}.");
+                    logger.Info($"[WamBroker] wamAccount.UserName: {wamAccount.UserName}.");
```

Comment:
This is PII

Reply:
`logger.InfoPii(
$"[WamBroker] wamwamAccount.UserName: {wamAccount.UserName}.",
$"[WamBroker] wamAccount.UserName: {string.IsNullOrEmpty(wamAccount.UserName)}." );